libs/transactions/src/lib/extensions/transaction-addition-details.ts

```diff
@@ -0,0 +1,13 @@
+import {InjectionToken, Type} from "@angular/core";
+import { TransactionItem } from "@backbase/data-ang/transactions";
+import {ExtensionComponent} from "../extension-slot.directive";
+
+export type TransactionAdditionalDetailsComponent = ExtensionComponent<TransactionItem>;
+
+export interface TransactionsJourneyExtensionsConfig {
+  transactionItem?: Type<TransactionAdditionalDetailsComponent>;
+  // other extensions slots here
+}
+
+// Private injection token
+export const ΘTRANSACTION_EXTENSIONS_CONFIG = new InjectionToken<TransactionsJourneyExtensionsConfig>('TRANSACTION_EXTENSIONS_CONFIG');
```

Comment:
is is common (for us) to mark variables private with `Θ` ?

Reply:
Some teams in BB do it, but not many. It's a convention I like.

Reply:
i don't have a (very) strong opinion on this, besides preferring consistency

Reply:
Tbh, I just did it here to emphasise this injection token does not need to be exported and customers do not need to manually create a provider for it as it's all handled internally to the journey in a type-safe way.